etna/bin/etna

```diff
@@ -1,12 +1,11 @@
 #!/usr/bin/env ruby
 
-require 'bundler'
```

Comment:
Keeping bundler means I can only run the gem in a directory with a `Gemfile` ...? Doesn't seem necessary for the basic project commands.

Reply:
Correct, also because the gem installation will have our dependencies installed in the system and not the bundle directory.
This line is most useful, however, for development purposes since it allows us to run without needed to perform the gem install each time.  Maybe we can conditionalize it somehow?

Reply:
Ah, I see. What would you conditionalize on, like an environment variable?

Reply:
Yeah.  We can add an environment variable in the Dockerfile that won't effect local ge installations.

Reply:
@corps, I've set it to only require if `ETNA_GEM_DEVELOPMENT` is set in the environment.